### PR TITLE
Add Chart datum Norway vertical grid.

### PR DESCRIPTION
**Pull request by himsve**
This is the grid with vertical shifts for the transformation from ETRS89 ell. heights to Chart Datum Norway depths.

As I can see from https://github.com/OSGeo/PROJ/tree/master, the transformation by grids from ellipsoidal heights to ortometric depths is not yet solved (regarding the new method code EPSG:1110: https://github.com/OSGeo/PROJ/blob/78eee957b0eff8b74df8a1f0e60a95355e7fa1c6/src/iso19111/operation/transformation.cpp#L2013).

The transformation from from ETRS89 ell. heights to CD Norway depths (EPSG:4937>EPSG:9672) is temporary solved in a personal branch of Proj: https://github.com/himsve/PROJ/commits/typo_cdnorway. The branch is only workaround for internal use and would not be pull requested. 

There is also an issue in Proj where the transformation method above is discussed: https://github.com/OSGeo/PROJ/issues/2739

### CODE REVIEW

**Comment by rouault** [comment]
The Computed Min/Max=-959.493,45.875 reported on the dataset looks strange to me.  The max value seems consitent of the geoid ondulation of no_kv_href2008a.tif, but -959 is suspicious to me

**Comment by himsve** [comment]
@rouault Yes, I have seen the strange values in edge of the grid. The values are adopted from the original GravSoft bin file.

But, if you like, it is an easy job for me the remove extreme values in the grid.

**Comment by rouault** [comment]
> But, if you like, it is an easy job for me the remove extreme values in the grid.

Perhaps better if not too complicated. %y remark was just to make sure nothing wrong had happened

**Comment by himsve** [comment]
The grid is cleaned in the last commit.

File: copyright_and_licenses.csv [hunk 1]
```diff
@@ -170,6 +170,7 @@ nl_nsgi_nlgeo2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI)
 nl_nsgi_rdcorr2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),CC-BY-4.0,,
 nl_nsgi_rdtrans2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),CC-BY-4.0,,
 nl_nsgi_README.txt,Disclaimed,Public domain,,
+no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,2021a,
```
Comment (rouault) [inline copyright_and_licenses.csv:173]: the version_added should be the version number of the PROJ-data package, so 1.9
```suggestion
no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,1.9,
```

**Pull request merged**